src/undo_log/undo_log.c

```diff
@@ -480,9 +480,9 @@ void UndoLog_RemoveLabels
 // undo schema addition
 void UndoLog_AddSchema
 (
-	UndoLog *log,                // undo log
-	int schema_id,               // id of the schema
-	SchemaType t                 // type of the schema
+	UndoLog *log,  // undo log
+	int schema_id, // id of the schema
```

Comment:
at the moment the "convention" is 2 spaces between code and the beginning of a comment.

Reply:
Purely aesthetics.
As long as we don't have a formatter in place, we'll have to try and follow these unwritten rules